README.md

```diff
@@ -109,7 +109,15 @@ $ npm start
 
 # Conversation Slots Discussion
 
-The power of Slots is in how it reduces the numbe of nodes requiered to implement logic in your Watson Conversation Dialog. Open up the Dialog, and we'll have a look:
+The power of Slots is in how it reduces the number of nodes requiered to implement logic in your Watson Conversation Dialog. Here's a partial conversation Dialog using the old methods:
+
+![](doc/source/images/pizzaOldWay.png)
+
+And here's a more complete Dialog using slots, which puts all the logic in one Node!
```

Comment:
nit?: Probably should take a look at cap style.  I see "Node" here but "nodes" above. I wouldn't capitalize Node. But I also see "Dialog" capitalized pretty consistently.  It's a British style thing, I think. Maybe choose an approach and do a quick scan (not sure but perhaps lowercase nodes but upper Node is intentional old chap). This is just the one case that kind of stood out to me.